ccf/tests/test_urls.py

```diff
@@ -83,3 +83,22 @@ def test_post_job_when_not_logged_in_redirects(self):
         SimpleTestCase().assertRedirects(
             response, "/accounts/login/?next=/jobs/create/"
         )
+
```

Comment:
using something along the lines of 
```
@patch("app.views.jobs_view.render")
def test_jobs_lists_are_correct(render_mock):
    # Test code here
    render_mock.assert_called_with(dict_with_correct_values)
```

We can probably test here that the lists being returned definitely contain only jobs for that user, and that all the correct jobs are in the correct places. Might not be worth the time with the due date as soon as it is, but definitely worth thinking about.